verifier/fake/fakeverifier.go

```diff
@@ -54,6 +77,7 @@ func (fc *fakeClient) VerifyAttestation(_ context.Context, req verifier.VerifyAt
 			Issuer:    "https://confidentialcomputing.googleapis.com/",
 			Subject:   "https://www.googleapis.com/compute/v1/projects/fakeProject/zones/fakeZone/instances/fakeInstance",
 		},
+		MachineStateMarshaled: string(msJSON),
```

Comment:
nit: could you update the comments on this method? The returned claim is no longer hardcoded.